frontend/components/Header/_index.html.erb

```diff
@@ -0,0 +1,3 @@
+<header class="Header">
+  <%= image_tag asset_pack_path('components/Header/static/logo.svg'), :class => "Header-logo" %>
```

Comment:
Can you add an `alt` attribute on this image.

We should also favour the newer way of writing hashes.

``` diff
- :class => "Header-logo"
+ class: "Header-logo"
```